bills.csv

```diff
@@ -0,0 +1,2788 @@
+2020,3_4,SR-4,foo,JOHN W. FONFARA, Y 

```

Comment:
I don't think we actually want this checked in

Reply:
oh, are you checking it in because we want to be able to write this and share it out to people so that non-programmers can use this dataset? We had a conversation about this and I forgot about that until just now

Reply:
regardless, this data seems off, every row has `foo`, and there's no header row

Reply:
Yeah we don't need this right now, I just wrote to a csv before writing to the database to make sure I understood the process. I forgot to delete it